examples/common-cxx/meson.build

```diff
@@ -0,0 +1,8 @@
+libexample_commoncxx = static_library('common', 'scpi-def.cpp',
+  include_directories : [ inc, include_directories('.') ],
```

Comment:
Since meson 0.50 (you specified an 0.60 minimum requirement), the arguments to `include_directories: [...]` can be a string, and it is automatically converted as though the `include_directories()` function was used. The function is still useful for e.g. defining the `inc` variable in one directory and using it in another. :)

That being said, `implicit_include_directories: true` is the default and means that executables/libraries already, implicitly, have `'.'` as an include directory. So this is redundant.

(You still need it down below for declare_dependency. :))